docs/wiki/ALZ-Contribution-Guide.md

```diff
@@ -0,0 +1,183 @@
+
+![Example policy def in initiative](./media/example-def-in-init.png)
+
+Also find it in the policyDefinitions and remove reference as well:
+
+![Example policy def in initiative 2](./media/example-def-in-init-2.png)
+
+When working within the policy files, to read parameters which are set at the top level of the policy definition a double escape is needed for ARM. So instead of using `[parameters('someParameter')]` within the policy, you should use `[[parameters('someParameter')]` instead.
+
+> **Note:** When testing the policy manually in the portal or another deployment outside of the ALZ Accelerator (Portal), you will need to remove the double escaping, `[[`, and revert to normal ,`[`'
+
+When working with policies that are assigned by default, these are located under the [eslzArm/managementGroupTemplates/policyAssignments](../../eslzArm/managementGroupTemplates/policyAssignments) folder. References to policy definitions are done through the assignments, so if any amendments are done to default assigned policies, they should be amended here too. A wiki to default assignments can be found [in the wiki](./ALZ-Policies.md).
+
+Policies in `eslzArm.json` file will also need updating if wanting to assign a new policy that is located. The file for this amendment [is located here](../../eslzArm/eslzArm.json).
+
+Details of 
+
+### Contribution scope
+
+The following is the scope of contributions to this repository:
+
+As the Azure platform evolves and new services and features are validated in production with customers, the design guidelines will be updated in the overall architecture context.
+
+With new Services, Resources, Resource properties and API versions, the implementation guide and reference implementation must be updated as appropriate.
+Primarily, the code contribution would be centered on Azure Policy definitions and Azure Policy assignments for the reference implementation.
+
+Submit a pull request for documentation updates using the following template 'placeholder'.
+
+### How to submit Pull Request to upstream repo
+
+1. Create a new branch based on upstream/main by executing following command
+
+    ```shell
+    git checkout -b feature upstream/main
+    ```
+
+2. Checkout the file(s) from your working branch that you may want to include in PR
+
+    ```shell
+    #substitute file name as appropriate. below example
+    git checkout feature: .\.docs\Deploy\Deploy-lz.md
+    ```
+
+3. Push your Git branch to your origin
+
+    ```shell
+    git push origin -u
+    ```
+
+4. Create a pull request from upstream to your remote main
+
+## Code of Conduct
```

Comment:
We should probably also update this to point to the standard [Microsoft code of conduct](https://github.com/Azure/Enterprise-Scale/blob/main/CODE_OF_CONDUCT.md) page?

Reply:
amended